cpp/modmesh/pilot/R3DWidget.cpp

```diff
@@ -84,8 +84,31 @@ void R3DWidget::updateWorld(std::shared_ptr<WorldFp64> const & world)
             child->deleteLater();
         }
     }
+
     new RVertices(world, m_scene);
     new RLines(world, m_scene);
+
+    auto vertices = std::make_shared<RVertices>(world, m_scene);
```

Comment:
Use shared pointer to prevent the `error: Potential leak of memory pointed to by 'vertices'`.

Reply:
Qt objects may not use shared pointer but the Qt object tree: https://doc.qt.io/qt-6.8/objecttrees.html .  In short, you need to set parent properly.